zuul-core/src/test/java/com/netflix/zuul/message/http/HttpRequestMessageImplTest.java

```diff
@@ -239,4 +241,13 @@ public void testCleanCookieHeaders() {
         assertEquals(" BlahId=12345; something=67890;", HttpRequestMessageImpl.cleanCookieHeader(" Secure, BlahId=12345; Secure, something=67890;"));
         assertEquals("", HttpRequestMessageImpl.cleanCookieHeader(""));
     }
+
+    @Test
+    public void shouldPreferClientDestPortWhenInitialized() {
```

Comment:
Line wrapping is odd here.    Either fill all the lines all the way, or put one arg per line.  This is for consistency.

Reply:
Yes, I'll actually reformat with our code style formatter here.